### ISSUE TITLE
Unable to store m_flFlashMaxAlpha

### ISSUE THREAD

**Issue by BaiFeng666** [86hrs/cheat-menu#2]
`
 if (!health) continue;

		const auto baseFlashAlpha = mem.Read<float>(localPlayer + offsets::m_flFlashMaxAlpha);
    // ----------BHOP----------
    const auto flags = mem.Read<int32_t>(localPlayer + offsets::m_fFlags);
    if (globals::bhop)
@@ -112,6 +114,8 @@ void hacks::miscThread(const Memory &mem) noexcept {
    // ----------Ignore Flash----------
    if (globals::ignoreFlash)
      mem.Write(localPlayer + offsets::m_flFlashMaxAlpha, 0.f);
		else
			mem.Write(localPlayer + offsets::m_flFlashMaxAlpha, baseFlashAlpha);
`
It's in an update method,Imagine that you changed this m_flFlashMaxAlpha,after you read m_flFlashMaxAlpha is still the you changed value.

You should set a global variable store and only store it once (not in a loop) before starting to modify Flash.

**Comment by BaiFeng666** [comment]
我英语不好 希望你能看懂...

**Comment by 86hrs** [comment]
yes I know, maybe declare the variable before the gui running loop

**Comment by 86hrs** [comment]
> 我英语不好 希望你能看懂...

No problem :D

**Comment by 86hrs** [comment]
Fixed it, used a different offset
